\chapter{Conclusions}
The chordate body plans is fairly conserved throughout the phyla, with most developing into a tadpole larvae containing a hollow dorsal neural tube, and a postanal tail containing a notochord flanked by muscle cells. Molguilds are especially useful models because for studying the development of body plans because they have both tailed and tail-less species, two of which has the ability to produce interspecific hybrids. This allows us to examine the mechanisms of evolutionary divergent body plans at an allele-specific level. 

\section{Evaluating a lightweight transcriptome assembly pipeline on two closely related ascidian species}
Many non-model systems are now being sequenced with the drop in sequencing price. The methodology for assembly is not cut and dry, there are number step\textemdash quality trimming, filtering, choice of assembler and metrics to evaluate assembly. One of the standard is the N50, which has been designed for genomes and does not clearly translate to transcriptomes because of their fragmented nature, and the likelihood of chimeric contigs. We show that pre filtering quality trimmed assembly reads does not reduce the information content of the assembly. Many times assembly methods are chosen on the usability of software and the availability of resources. We have demonstrated that the Oases and Trinity assemblers returns similar results, suitable for downstream analysis. Our pipelines are available so we also provide methodology to be used by future researchers. 

\subsection{Contributions}
The experimental design for the RNA sequence analysis was decided by C. Titus Brown (CTB) and Billie J. Swalla (BJS). The fertilization and collection of RNA samples were done by BJS and myself. Library preparation for Illumina sequencing was done by Kanchan Pavangadkar. All downstream analysis was done by myself. The idea for evaluating de novo assembly pipeline came about through conversations with CTB, as well as methods for evaluation the transcriptome assemblies. Writing was done by me with edits from BJS and CTB.

\section{Genome assembly and characterization}
\subsection{Contributions}
The experimental design for the DNA sequencing was decided by CTB, BJS, and Lionel Christiaen (LC). Embryonic samples were collected by BJS, LC, Alberto Stolfi (AS), Claudia Racioppi (CR) and myself. Genome assembly was conducted by me. Ideas for examining divegand were developed by AS, orthologous sequences were identified through the use of RBH blast and done so by me because the sequences were not yet search about on the Aniseed database. The \textit{Hox} analysis was done by me and was conduced because of a question from David Arnosti. Writing was done by AS, myself with edits from BJS, CTB, CR, and LC.

\section{Differential expression analysis of tail loss in an invertebrate chordate}
Studying closely related organisms gives us insight into the underlying evolutionary mechanisms of divergence and underlying development. Indirect and direct developing sea urchins have been studied showing that change in axis and cell fate leads to divergent larval body plans that exhibits similar adults \cite{}. In contrast, ascidians retain their typical cell division and invariant cell fate. This is evident in both direct and indirect tail-less ascidians \cite{}. 
\subsection{Contributions}
The experimental design is the same for the "Evaluating a lightweight transcriptome assembly pipeline on two closely related ascidian species" chapter (3). Transcript models were assembled by me. Read mapping for differential expression analysis was conducted by me, and the analysis of hybrid expression counts were the idea of CTB. The differential expression analysis was conducted by me, including the idea to focus on the overlapping upregulated gene in \textit{M. oculata} and the interspecific hybrid. Annotation of the overlapping upregulated genes was done by AS, Anna Di Gregorio and myself. Writing was done by me with edits from AS, CR, and CTB.
  
Next generation sequencing is a great way to study non-model species. A broad swath of information can be gained from both RNA and DNA sequencing. This technology has allowed us to analyzed two closely related invertebrate chordates that also hybridize. This study has given us insight into the mechanisms behind tail-loss and development. Gene loss has occurred but other factors appear to be involved in tail-loss because expression from the tailed species appears to recover the loss features. 
